src/utils/calculateDate.js

```diff
@@ -0,0 +1,10 @@
+import { formatedDate } from "./formatedDate"
+
+export const calculateDate = (date, newDate) => {
+    const oneDayMilliseconds = 24 * 60 * 60 * 1000
+    const start = new Date(date)
+    const end = new Date(formatedDate(newDate))
+    const timeDifference = end - start
+    const daysPassed = Math.floor(timeDifference / oneDayMilliseconds)
+    return `${daysPassed}`
+}
```

Comment:
@umh-wtag Apu, I guess this change is related to our [Step-4: Add functionality to make a task state to done](https://trello.com/c/3p3nj18C). Isn't it? It yes to keep things well separated me may have it on that PR. Thanks.

Reply:
bhaia i have removed this part since it's a change related to step-4.